src/EditorFeatures/Core/GoToDefinition/IGoToDefinitionService.cs

```diff
@@ -24,4 +24,9 @@ internal interface IGoToDefinitionService : ILanguageService
         /// <returns>True if navigating to the definition of the symbol at the provided position succeeds.  False, otherwise.</returns>
         bool TryGoToDefinition(Document document, int position, CancellationToken cancellationToken);
     }
+
+    internal interface IAsyncGoToDefinitionService : IGoToDefinitionService
```

Comment:
So a bit confused if this inheritance makes sense -- this is only bringing along two methods, one of which is the sync one (and not implemented), and then also the other FindDefinitions which returns a INavigableItem -- how are we thinking about those two ultimately merging?

Reply:
done.